t/variation.t

```diff
@@ -210,7 +210,6 @@ my $publication_output =
   is($variants_json->[0]->{MAF},$publication_output->[0]->{MAF} ,'PMID MAF key'); # Interestingly these are not numerically equal under certain testing conditions...
   is($variants_json->[0]->{ambiguity}, $publication_output->[0]->{ambiguity} ,'PMID ambiguity key');
   is($variants_json->[0]->{var_class}, $publication_output->[0]->{var_class} ,'PMID source key');
-  ok(!$variants_json->[0]->{ancestral_allele},'PMID ancestral_allele key not set');
```

Comment:
is this supposed to be removed?

Reply:
This is also removed in #404

Reply:
weird, don't know why github shows it as a change...